docs/changelog/87773.yaml

```
@@ -0,0 +1,5 @@
+pr: 87773
+summary: Add setting for keep-alive duration for oidc back-channel
```

Since we're now enabling this by default, I think a better summary would be
```suggestion
summary: Automatically close idle connections in OIDC back-channel
```